config/config.py

```diff
@@ -13,7 +15,11 @@ class Config:
     WORKFLOWS = 'workflows'
     CONFIG_FILE_NAME = 'config.yml'
 
-    def __init__(self, config_dir: str, profiles_dir: str, profile_target: str = None) -> None:
+    DEFAULT_CONFIG_DIR = Path.home() / '.edr'
+    DEFAULT_PROFILES_DIR = Path.home() / '.dbt'
```

Comment:
Are you sure Path is supported on all our currently supported python version?

Reply:
Yes, `pathlib` was introduced in Python 3.4.